certinel.go

```diff
@@ -86,6 +99,17 @@ func (c *Certinel) Close() (err error) {
 	return err
 }
 
+// Wait blocks until the intial load of the key pair completes. If it returns
+// a nil error, GetCertificate and GetClientCertificate are ready for use.
+func (c *Certinel) Wait() error {
+	if c.loaded == nil {
+		return nil
+	}
+
+	<-c.loaded
+	return c.err
+}
+
 // GetCertificate returns the current tls.Certificate instance. The function
```

Comment:
Probably worth mentioning this can return `nil` has not yet been loaded (or has been unloaded)